docs/source/tutorials/quickstart.rst

```diff
@@ -2,145 +2,150 @@
 Quickstart
 ==========
 
-This document will lead you step by step through the necessary procedure to get
-started with manim for the first time as soon as possible.  This tutorial
-assumes you have already installed manim following the steps in
-:doc:`../installation`.
+.. note::
+ Before proceeding, install manim and make sure it's running properly by
+ following the steps in :doc:`../installation`. For
+ information on using manim with Jupyterlab or Jupyter notebook, go to the
+ documentation for the
+ :meth:`IPython magic command <manim.utils.ipython_magic.ManimMagic.manim>`,
+ ``%%manim``.
 
+Overview
+*************************************************
 
-Start a new project
-*******************
+This quickstart guide will lead you through creating a sample project using manim: an animation
+engine for precise programmatic animations.
 
-To start a new manim video project, all you need to do is choose a single
-folder where all of the files related to the video will reside.  For this
-example, this folder will be called ``project``,
+First, you will use a command line
+interface to create a ``Scene``, the class through which manim generates videos.
+In the ``Scene`` you will animate a circle. Then you will add another ``Scene`` showing
+a square transforming into a circle. This will be your introduction to manim's animation ability.
+Afterwards, you will position multiple manim objects (mobjects). Finally, you
+will learn the syntax of ``.animate``, a powerful feature that displays how
+mobjects change when you modify them with methods.
```

Comment:
```suggestion
will learn the syntax of ``.animate``, a powerful feature that animates
Mobjects' transformations when you modify them with methods.
```

Reply:
Could probably still be worded a little better.

Reply:
How about removing the part about transformations (it's explained in the portion below anyway) and for the overview go with something like

> Finally, you will learn the ``.animate`` syntax, a powerful feature that animates the methods you use to modify Mobjects.

Reply:
That sounds good to me!